setup.py

```diff
@@ -13,6 +13,7 @@
 
 REQUIREMENTS_TEST = open(os.path.join(HERE, "requirements_test.txt")).readlines()
 REQUIREMENTS = [
+    "smart_open",
```

Comment:
actually `smart_open[http,s3]` but both boto3 and requests are already there so all good